### DISCUSSION TITLE
通过nodejs服务器欺骗更新

### DISCUSSION THREAD

**Discussion by wyxdlz54188** [PenUniverse/PenMods#260] [你问我答]
奇技淫巧里有个通过nodejs服务器欺骗更新的教程中有个，
修改host系统文件
不知道改哪里
![Uploading Notepad_202410282109_05777.png…]()

**Comment by SkySight-666** [comment]
`C:\Windows\System32\drivers\etc\hosts`
记得使用电脑热点
将`iotapi.abupdate.com`重定向到你自己的欺骗服务器

**Comment by wyxdlz54188** [comment]
怎么打包img工具一直报错

**Reply by wyxdlz54188** [reply]
如果方便的话帮忙改下rootfs的密码并打包一下呗，我这试了几个工具一直报错
全量包
http://iotdown.mayitek.com/1700649481/13961985/4b11d8e6-9c4f-403a-b106-5d6c737a6304.img

**Reply by wyxdlz54188** [reply]
![Notepad_202410282109_05777](https://github.com/user-attachments/assets/de71ad37-ae8d-49c1-81be-a3b611880b2b)

**Reply by SkySight-666** [reply]
> 怎么打包img工具一直报错

首先如果主控不是rk无解打包器

其次我原贴中提到的是使用`sed -i` 命令替换MD5

第三是要在模拟请求时，改版本号为99.99.90来post到rec模式下的全量包；这个文件一般较大。

**Reply by wyxdlz54188** [reply]
我改了全量包就500mb

**Comment by wyxdlz54188** [comment]
首先我确定A6Pro是rk因为rkTool可以读取模式但是无法写入
其次md5密码替换后经测试无法登入
第三rootfs.img挂载后是只读的
第四网上资料已经翻遍了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月1日(周五) 晚上11:00
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260)




  
怎么打包img工具一直报错
  
首先如果主控不是rk无解打包器
 
其次我原贴中提到的是使用sed -i 命令替换MD5
 
第三是要在模拟请求时，改版本号为99.99.90来post到rec模式下的全量包；这个文件一般较大。

**Reply by SkySight-666** [reply]
> 首先我确定A6Pro是rk因为rkTool可以读取模式但是无法写入 其次md5密码替换后经测试无法登入 第三rootfs.img挂载后是只读的 第四网上资料已经翻遍了
> […](#)
> ---原始邮件--- 发件人: ***@***.***&gt; 发送时间: 2024年11月1日(周五) 晚上11:00 收件人: ***@***.***&gt;; 抄送: ***@***.******@***.***&gt;; 主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260) 怎么打包img工具一直报错 首先如果主控不是rk无解打包器 其次我原贴中提到的是使用sed -i 命令替换MD5 第三是要在模拟请求时，改版本号为99.99.90来post到rec模式下的全量包；这个文件一般较大。 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

首先你应该binwalk看看里面有没有 `adb_shell_auth.sh`
全量500m实在不太可能，我看看吧

**Comment by wyxdlz54188** [comment]
***@***.***
私信我



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月4日(周一) 晚上10:27
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260)




  
首先我确定A6Pro是rk因为rkTool可以读取模式但是无法写入 其次md5密码替换后经测试无法登入 第三rootfs.img挂载后是只读的 第四网上资料已经翻遍了
 …
 ---原始邮件--- 发件人: @.&gt; 发送时间: 2024年11月1日(周五) 晚上11:00 收件人: @.&gt;; 抄送: @.@.&gt;; 主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260) 怎么打包img工具一直报错 首先如果主控不是rk无解打包器 其次我原贴中提到的是使用sed -i 命令替换MD5 第三是要在模拟请求时，改版本号为99.99.90来post到rec模式下的全量包；这个文件一般较大。 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: @.***&gt;
  
首先你应该binwalk看看里面有没有 adb_shell_auth.sh
 全量500m实在不太可能，我看看吧

**Reply by SkySight-666** [reply]
> ***@***.*** 私信我
> […](#)
> ---原始邮件--- 发件人: ***@***.***&gt; 发送时间: 2024年11月4日(周一) 晚上10:27 收件人: ***@***.***&gt;; 抄送: ***@***.******@***.***&gt;; 主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260) 首先我确定A6Pro是rk因为rkTool可以读取模式但是无法写入 其次md5密码替换后经测试无法登入 第三rootfs.img挂载后是只读的 第四网上资料已经翻遍了 … ---原始邮件--- 发件人: @.&gt; 发送时间: 2024年11月1日(周五) 晚上11:00 收件人: @.&gt;; 抄送: @.@.&gt;; 主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260) 怎么打包img工具一直报错 首先如果主控不是rk无解打包器 其次我原贴中提到的是使用sed -i 命令替换MD5 第三是要在模拟请求时，改版本号为99.99.90来post到rec模式下的全量包；这个文件一般较大。 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: @.***&gt; 首先你应该binwalk看看里面有没有 adb_shell_auth.sh 全量500m实在不太可能，我看看吧 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

如何 
我没有你的电子邮件地址

**Comment by wyxdlz54188** [comment]
QQ。 3759823756

---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月4日(周一) 晚上11:50
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260)




  
@.*** 私信我
 …
 ---原始邮件--- 发件人: @.&gt; 发送时间: 2024年11月4日(周一) 晚上10:27 收件人: @.&gt;; 抄送: @.@.&gt;; 主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260) 首先我确定A6Pro是rk因为rkTool可以读取模式但是无法写入 其次md5密码替换后经测试无法登入 第三rootfs.img挂载后是只读的 第四网上资料已经翻遍了 … ---原始邮件--- 发件人: @.&gt; 发送时间: 2024年11月1日(周五) 晚上11:00 收件人: @.&gt;; 抄送: @.@.&gt;; 主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260) 怎么打包img工具一直报错 首先如果主控不是rk无解打包器 其次我原贴中提到的是使用sed -i 命令替换MD5 第三是要在模拟请求时，改版本号为99.99.90来post到rec模式下的全量包；这个文件一般较大。 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: @.&gt; 首先你应该binwalk看看里面有没有 adb_shell_auth.sh 全量500m实在不太可能，我看看吧 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: @.&gt;
  
如何
 我没有你的电子邮件地址

**Comment by wyxdlz54188** [comment]
QQ已发你，邮件太麻烦了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月4日(周一) 晚上11:50
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260)




  
@.*** 私信我
 …
 ---原始邮件--- 发件人: @.&gt; 发送时间: 2024年11月4日(周一) 晚上10:27 收件人: @.&gt;; 抄送: @.@.&gt;; 主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260) 首先我确定A6Pro是rk因为rkTool可以读取模式但是无法写入 其次md5密码替换后经测试无法登入 第三rootfs.img挂载后是只读的 第四网上资料已经翻遍了 … ---原始邮件--- 发件人: @.&gt; 发送时间: 2024年11月1日(周五) 晚上11:00 收件人: @.&gt;; 抄送: @.@.&gt;; 主题: Re: [PenUniverse/PenMods-release] 通过nodejs服务器欺骗更新 (Discussion #260) 怎么打包img工具一直报错 首先如果主控不是rk无解打包器 其次我原贴中提到的是使用sed -i 命令替换MD5 第三是要在模拟请求时，改版本号为99.99.90来post到rec模式下的全量包；这个文件一般较大。 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: @.&gt; 首先你应该binwalk看看里面有没有 adb_shell_auth.sh 全量500m实在不太可能，我看看吧 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: @.&gt;
  
如何
 我没有你的电子邮件地址

**Reply by LuorixDev** [reply]
请问有结果了吗，我也是S6 Pro的

**Reply by wyxdlz54188** [reply]
还没，我托我同学去破解了下周一出结果，先加QQ

**Reply by wang-lu-yuan** [reply]
> 还没，我托我同学去破解了下周一出结果，先加QQ

现在出结果了吗？~~不会没出吧~~

**Reply by wyxdlz54188** [reply]
出了

**Reply by wyxdlz54188** [reply]
https://posc.dedyn.io/
